### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
```
Comment (gabro) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
name: 🐛 Bug report
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: Feature request
```
Comment (gabro) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
name: 🚀 Feature request
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (gabro) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
labels: 'bug'
```

**Comment by olafurpg** [comment]
Merged into master manually.

**Pull request closed without merging**